packages/editor/src/components/block-bindings/bindings-ui.js

```diff
@@ -0,0 +1,278 @@
+/**
+ * WordPress dependencies
+ */
+import { useState, cloneElement, Fragment } from '@wordpress/element';
+import {
+	BlockControls,
+	updateBlockBindingsAttribute,
+} from '@wordpress/block-editor';
+import {
+	Button,
+	createSlotFill,
+	MenuItem,
+	MenuGroup,
+	Popover,
+} from '@wordpress/components';
+import {
+	plugins as pluginsIcon,
+	chevronDown,
+	chevronUp,
+} from '@wordpress/icons';
+import { addFilter } from '@wordpress/hooks';
+
+const blockBindingsWhitelist = {
```

Comment:
Let's replace the variable name to `blockBindingsSupportedAttributes`. "whitelist" has some negative connotation and we try to avoid it in the WordPress projects.

Reply:
I'm really sorry for that. Thanks for pointing it out. I'll change it.